Networking/Networking/Remote/GenerativeContentRemote.swift

```diff
@@ -199,74 +199,62 @@ private extension GenerativeContentRemote {
                            categories: [ProductCategory],
                            tags: [ProductTag],
                            token: JWToken) async throws -> AIProduct {
+        let input = [
+            "You are a WooCommerce SEO and marketing expert, perform in-depth research about the product " +
+            "using the provided name, keywords and tone.",
+            "Your response should be in JSON format and don't send anything extra.",
```

Comment:
IMO we should be more specific about this to avoid the cases where [some fields are nested](https://github.com/woocommerce/woocommerce-ios/issues/11188#issuecomment-1841537724):

```suggestion
            "Your response should be in a valid JSON format. Each field should contain a value in String, no nested dictionaries are accepted.",
```

Reply:
Unfortunately, we cannot use this as we expect a nested JSON object for `shipping` field. 

Did you face the nested issue again after this PR's changes?

Reply:
I understand, please ignore my suggestion for asking no nested dictionaries then. I cannot consistently reproduce the nested issue, I hope the update for language prompts helps with that.